### PR TITLE
10-14 changelog

### PR DESCRIPTION
**Pull request by melrchen**

### CODE REVIEW

File: changelog.mdx [hunk 1]
```diff
@@ -7,6 +7,39 @@ description: Updates to Chalk!
 
 ---
 
+## October 14, 2024
+
+### SQL explorer in dashboard for datasets
+
+You can now run SQL queries in the dashboard to explore data in [datasets](https://docs.chalk.ai/docs/datasets).
```
Comment (hugokitano) [inline changelog.mdx:14]: maybe 'You can now run SQL queries on [dataset] outputs on the Chalk dashboard to explore your data.

File: changelog.mdx [hunk 2]
```diff
@@ -7,6 +7,39 @@ description: Updates to Chalk!
...
+
+### Optionally evict nulls from your DynamoDB online store
+
+Last week we enabled the option to decide whether to persist null values for features in Redis lightning
+online stores, and this week we have enabled this feature in DynamoDB online stores. By default, null
+values are persisted in the online store for features defined as `Optional`, but you can set `cache_nulls=False`
+in the `feature` method to evict null values. Read more about how to use the `cache_nulls` parameter
+[here](/api-docs#feature.cache_nulls).
+
+### Set environment variables and more in Advanced section of Cloud Resource Configurations page in dashboard
```
Comment (hugokitano) [inline changelog.mdx:28]:
```suggestion
### Set environment variables and more in the Advanced section of the Cloud Resource Configurations page on the dashboard
```

File: changelog.mdx [hunk 3]
```diff
@@ -7,6 +7,39 @@ description: Updates to Chalk!
...
+
+You can set cloud resource configurations for your environment by navigating to `Settings > Resources` in the
```
Comment (hugokitano) [inline changelog.mdx:30]:
```suggestion
You can set cloud resource configurations for your environment by navigating to `Settings > Resources` on the
```

File: changelog.mdx [hunk 4]
```diff
@@ -7,6 +7,39 @@ description: Updates to Chalk!
...
+dashboard. In addition to specifying resource configurations for resource groups like instance counts and CPU,
+you can now also set environment variables and other settings like Kubernetes Node Selectors. A notable
```
Comment (hugokitano) [inline changelog.mdx:32]:
```suggestion
you can now also set environment variables and other settings like Kubernetes Node Selectors. One notable
```

File: changelog.mdx [hunk 5]
```diff
@@ -7,6 +7,39 @@ description: Updates to Chalk!
...
+environment variable that you can set is `AUTO_SHUTDOWN_PERIOD`, which should be set to the number of seconds
+of inactivity after which the branch server is automatically shut down. The Kubernetes Node Selector allows
```
Comment (hugokitano) [inline changelog.mdx:34]:
```suggestion
of inactivity after which the branch server will be automatically shut down. The Kubernetes Node Selector enables
```

File: changelog.mdx [hunk 6]
```diff
@@ -7,6 +7,39 @@ description: Updates to Chalk!
...
+you to specify the machine family you would like to use for your deployment. For example, this would be [EC2
```
Comment (hugokitano) [inline changelog.mdx:35]:
```suggestion
you to specify the machine family you would like to use for your deployment. This would map to  [EC2
```

**Review by hugokitano — APPROVED** [review]

File: changelog.mdx [hunk 7]
```diff
@@ -7,6 +7,39 @@ description: Updates to Chalk!
...
+values are persisted in the online store for features defined as `Optional`, but you can set `cache_nulls=False`
+in the `feature` method to evict null values. Read more about how to use the `cache_nulls` parameter
+[here](/api-docs#feature.cache_nulls).
+
+### Set environment variables and more in Advanced section of Cloud Resource Configurations page in dashboard
+
+You can set cloud resource configurations for your environment by navigating to `Settings > Resources` in the
+dashboard. In addition to specifying resource configurations for resource groups like instance counts and CPU,
+you can now also set environment variables and other settings like Kubernetes Node Selectors. A notable
+environment variable that you can set is `AUTO_SHUTDOWN_PERIOD`, which should be set to the number of seconds
```
Comment (AndyMoreland) [inline changelog.mdx:33]: @melrchen I think we should perhaps omit discussion of this env var in the changelog -- it's kind of 80% baked & causes a fair amount of customer support questions w.r.t. "is my branch server on or not"?

File: changelog.mdx (hunk 2 above)
Comment (melrchen) [reply changelog.mdx:28]: Gonna try to stay consistent with "in the dashboard"

File: changelog.mdx (hunk 7 above)
Comment (melrchen) [reply changelog.mdx:33]: omitting!

**Pull request merged**